### PR TITLE
Feature/layout

### PR DESCRIPTION
**Pull request by sarisiza**
Created UI layouts

- Main Activity
- Bottom Navigation Menu
- Cocktails Details Fragment
- Ingredients View Holder
- Drink View Holder
- Filter View
- Search View

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -4,15 +4,23 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    tools:context=".MainActivity"
+    android:layout_margin="10dp">
 
-    <TextView
-        android:layout_width="wrap_content"
+    <androidx.appcompat.widget.SearchView
```
Comment (luizeth0) [inline app/src/main/res/layout/activity_main.xml:10]: Added a SearchView

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -4,15 +4,23 @@
...
-        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintEnd_toEndOf="parent"
+        android:queryHint="Search a Cocktail"/>
+
+<!--   TODO Add fragment container view-->
+
+    <com.google.android.material.bottomnavigation.BottomNavigationView
```
Comment (luizeth0) [inline app/src/main/res/layout/activity_main.xml:20]: Added a BottonNavigationView

File: app/src/main/res/layout/cocktails_details_fragment.xml [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (luizeth0) [inline app/src/main/res/layout/cocktails_details_fragment.xml:1]: Added a design in the Cocktail Details with ImageView, ImageButton, TextViews and RecyclerView

File: app/src/main/res/layout/drink_view_holder.xml [hunk 4]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (luizeth0) [inline app/src/main/res/layout/drink_view_holder.xml:1]: Created the drink_view_holder.xml layout for the Cocktails Recycler View

File: app/src/main/res/layout/filters_layout.xml [hunk 5]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (luizeth0) [inline app/src/main/res/layout/filters_layout.xml:1]: Added design on the filters layout

File: app/src/main/res/layout/ingredient_view_holder.xml [hunk 6]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (luizeth0) [inline app/src/main/res/layout/ingredient_view_holder.xml:1]: Created the ingredient_view_holder.xml layout for the ingredients Recycler View

File: app/src/main/res/menu/bottom_menu.xml [hunk 7]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (luizeth0) [inline app/src/main/res/menu/bottom_menu.xml:1]: Added items that will show it in the bottom menu

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -1,3 +1,18 @@
 <resources>
```
Comment (luizeth0) [inline app/src/main/res/values/strings.xml:1]: Defined all the values string in the app

**Review by luizeth0 — COMMENTED** [review]
Created Layout Resource Files
Added Icons on drawable folder

**Review by inaki92 — DISMISSED** [review]
LGTM!
Make sure to have latest changes form dev

**Review by rdhpn — DISMISSED** [review]
looks ok

**Review by luizeth0 — DISMISSED** [review]
LGTM!

**Review by luizeth0 — APPROVED** [review]
LGTM!

**Review by rdhpn — APPROVED** [review]
looks ok

**Review by inaki92 — APPROVED** [review]

**Pull request merged**